### ISSUE TITLE
Question about Lion

### ISSUE THREAD

**Issue by kellenf** [google/automl#1198]
Thank your for your paper and work,  but I meet some wired problem in training. Can you help me?
![image](https://github.com/google/automl/assets/19380638/1ed88f72-0f24-49ef-a6f1-748cf6f7645a)
It's my training process curve. Notice Lion will increase loss sometimes but decease later. It seems not stability? Can you give me some advises how to tune Lion's parameters?
Below is my setting:
Batchsize = 8, GPU = 64 A100, with syncBN
Network: PointPillars
Optimizer setting:
![image](https://github.com/google/automl/assets/19380638/6f580d1a-82a5-4fd8-83df-ad9b0807ff96)

**Comment by xiangning-chen** [comment]
Thanks for the data point. Can you please tell us your baseline settings? Like which optimizer, which learning rate and weight decay?

I think for instability problem, you can either reduce the learning rate a little bit or try to lower the beta2, for example, we used (0.95, 0.98) in our llm training.
